manifests/09-prometheus-rules-registry-operations.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: image-registry-rules
+  namespace: openshift-image-registry
+spec:
+  groups:
+  - name: imageregistry.operations.rules
+    rules:
+    - expr: |
+        label_replace(
+          label_replace(
+            sum by (operation) (imageregistry_request_duration_seconds_count{operation="BlobStore.ServeBlob"}), "operation", "get", "operation", "(.+)"
+          ), "resource_type", "blob", "resource_type", ""
+        )
+      record: imageregistry:operations_sum:count
```

Comment:
I think it should be `imageregistry:operations_count:sum`

Reply:
darn you're right, how did it end up like this

Reply:
fixed